src/seedu/addressbook/AddressBook.java

```diff
@@ -1151,9 +1151,24 @@ private static String getUsageInfoForExitCommand() {
      * @return  string without the sign
      */
     private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+        return removePrefix(s,sign);
     }
 
+    
+    /**
+    * Removes prefix from the given fullString if prefix occurs at the start of the string.
+    * 
+    * @param fullstring
```

Comment:
if you are not giving any additional information about the parameters, then you may as well remove the `@param` lines